volatility3/framework/plugins/linux/malware/malfind.py

```diff
@@ -37,6 +37,16 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                 element_type=int,
                 optional=True,
             ),
+            requirements.IntRequirement(
+                name="dump-size",
+                description="Amount of bytes to dump for each dirty region/page found - Default 64 bytes",
+                optional=True,
```

Comment:
```suggestion
                optional=True,
                default=64,
```
As mentioned below, the default parameter will ensure the value is properly set.